auth-web/src/components/auth/manage-business/ManageBusinessDialog.vue

```diff
@@ -0,0 +1,506 @@
+    this.isLoading = false
+    if (emitCancel) {
+      this.$emit('on-cancel')
+    }
+    this.dialog.show = false
+  }
+
+  formatBusinessIdentifier (): void {
+    this.businessIdentifierRules = [
+      v => !!v || 'Incorporation Number or Registration Number is required',
+      v => CommonUtils.validateIncorporationNumber(v) ||
+        'Incorporation Number or Registration Number is not valid'
+    ]
+    // this.businessIdentifier = CommonUtils.formatIncorporationNumber(this.businessIdentifier)
+  }
+
+  openHelp (): void {
+    this.$refs.helpDialog.open()
+  }
+
+  /** Emits event to parent initially and when business identifier changes. */
+  @Watch('businessIdentifier', { immediate: true })
+  private onBusinessIdentifierChange (): void {
+    this.$emit('on-business-identifier', this.businessIdentifier)
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import '$assets/scss/theme.scss';
+
+::v-deep {
+
+  .v-list-group{
+    border-bottom: 1px solid rgb(228, 228, 228);
+    &.top-of-list{
+      border-top: 1px solid rgb(228, 228, 228);
+    }
+    .item-content{
+      color: #000 !important;
+    }
+  }
+
+  .v-list-item{
+    background: #FFFFFF;
+    height: 4rem !important;
+    margin: 0 !important;
+  }
+
+  .v-list-item--link>
+  .v-list-item__title{
+    font-weight: 300 !important;
+    margin-left:-1rem !important;
+    color: var(--v-primary-base) !important;
+    .subtitle {
+      line-height: 1.5rem;
+      font-size: 9pt;
+      color: var(--v-primary-base) !important;
+      font-weight: normal;
+    }
+  }
+
+  .v-list-item--active>
+  .v-list-item__title{
+    font-weight: 600 !important;
+    margin-left:-1rem !important;
+    color: #000 !important;
+    .subtitle {
+      line-height: 1.5rem;
+      font-size: 9pt;
+      color: #000 !important;
+      font-weight: normal;
+    }
+  }
+
+  .v-list-item__content{
+    color: #000 !important;
+  }
+}
+
+// .add-business-unordered-list {
+//   list-style: none;
+//   padding-left: 1rem;
+
+//   li {
+//     margin-left: 1.5rem;
+
+//     &::before {
+//       content: "\2022";
+//       display: inline-block;
+//       width: 1.5em;
+//       color: $gray9;
+//       margin-left: -1.5em;
+//     }
+//   }
+// }
+
+// .underline-dotted {
+//   border-bottom: dotted;
+//   border-bottom-width: 2px;
+// }
+
+// .form__btns {
+//   display: flex;
+//   justify-content: flex-end;
+
+//   .v-btn + .v-btn {
+//     margin-left: 0.5rem;
+//   }
+
+//   #cancel-button,
+//   #add-button {
+//     min-width: unset !important;
+//     width: 100px;
+//   }
+
+//   // override disabled button color
+//   .v-btn[disabled]:not(.v-btn--flat):not(.v-btn--text):not(.v-btn--outlined) {
+//     color: white !important;
+//     background-color: $app-blue !important;
+//     opacity: 0.4;
+//   }
+// }
+
+// // remove whitespace below error message
+// .authorization {
+//   ::v-deep .v-text-field__details {
+//     margin-bottom: 0 !important;
+//   }
+// }
```

Comment:
Before the final commit/merge in this branch, obsolete code needs to be deleted, please.